### PR TITLE
London Class 7 - Murat Demirtas - JavaScript-Core-1-Week 1

### PR DESCRIPTION
**Pull request by muratdemirtas2001**
# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

### CODE REVIEW

File: exercises/E-strings-concatenation/exercise.js [hunk 1]
```diff
@@ -1,3 +1,9 @@
 // Start by creating a variable `message`
 
-console.log(message);
+let message="Hi everyone,nice to meet you, I am "
+
+let name="Murat"
```
Comment (kmh256) [inline exercises/E-strings-concatenation/exercise.js:5]: Needs a semi-colon ; at the end of the line of code

File: extra/1-currency-conversion.js [hunk 2]
```diff
@@ -15,7 +17,10 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(price) {
+  let newPrice = (price * 99) / 100;
```
Comment (kmh256) [inline extra/1-currency-conversion.js:21]: Suggestion: You could do price * 0.9;

**Review by kmh256 — COMMENTED** [review]
Great work Murat! I have added a couple of comments :)

File: extra/1-currency-conversion.js (hunk 2 above)
Comment (muratdemirtas2001) [reply extra/1-currency-conversion.js:21]: Thank you Kara, you have spotted every detail.I appreciate your support.

**Pull request closed without merging**